fix/__init__.py

```diff
@@ -140,6 +140,48 @@ def tol(self, x):
 
 class Database(object):
     def __init__(self):
+        pass
+
+
+
+def initialize():
+    thread = client.ClientThread('127.0.0.1', 3490)
+    thread.start()
+    thread.join()
+>>>>>>> 66b3f5f1bd8ac06e419f471d9394bd4f6e85ca79
```

Comment:
That's an early version of the file.  I think it was a merge conflict that I messed up on.  This is where I learned to actually try to run the program before committing the merge.  :-)